src/Orleans.Core.Abstractions/Orleans.Core.Abstractions.csproj

```diff
@@ -11,5 +11,9 @@
     <RootNamespace>Orleans</RootNamespace>
     <TargetFramework>netstandard1.3</TargetFramework>
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="1.1.1" />
```

Comment:
you can use the $(MicrosoftExtensionsDependencyInjection) for version number here. I think the whole sln is using that variable to keep the version number consistent

Reply:
Hmm..  Any idea how this variable is set?  It's not being used consistently, and I'm not sure using it at all is a good idea.  Changing a package version without going through the package manager can introduce many problems.  Will look into it more...

Reply:
it is set in Directory.Build.Props ... I see our csproj are using those. but I will @attilah here since he may know more about the suggested thing to do